src/features/user/ManageProjects/components/microComponent/ProjectSiteLayer.tsx

```diff
@@ -0,0 +1,42 @@
+import type { SitesGeoJSON } from '../../../../common/types/ProjectPropsContextInterface';
+
+import { Source, Layer } from 'react-map-gl-v7/maplibre';
+import themeProperties from '../../../../../theme/themeProperties';
+
+interface SiteLayerProps {
+  isSatelliteMode: boolean;
+  geoJson: SitesGeoJSON | null;
```

Comment:
Since we don't want to show this component when geoJson is null, update the prop types